src/blockchain_db/blockchain_db.cpp

```diff
@@ -35,17 +35,6 @@
 #include "ringct/rctOps.h"
 
 #include "lmdb/db_lmdb.h"
-#ifdef BERKELEY_DB
-#include "berkeleydb/db_bdb.h"
-#endif
-
-static const char *db_types[] = {
-  "lmdb",
-#ifdef BERKELEY_DB
-  "berkeley",
-#endif
-  NULL
-};
```

Comment:
I'd rather the selection mechanism stays. I've said that for quite a while, but since it seems nothing else is being added, maybe that could go. Anyone has an opinion here ?

Reply:
The general opinion is that it can go.